server/monitor/src/main/java/org/apache/accumulo/monitor/rest/api/AddlInformation.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.monitor.rest.api;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AddlInformation {
```

Comment:
I'll add the Javadoc, but this and some other classes similar to this are used to specify the JSON object of an Array, or as the root name for an object (Tablet Servers for example). I didn't find a better way to do it, if you know of one, I would appreciate if you point me in the right direction.